cmd/user-agent/src/pages/chapi/wallet/didcomm/mediator.js

```diff
@@ -96,9 +102,35 @@ export class AgentMediator {
         return response.invitation
     }
 
-    // TODO to be implemented
-    async requestDID() {
-        console.warn('feature for requesting peer DID from router is not yet available')
+    async requestDID(reqDoc) {
+        let res = await this.aries.mediator.getConnections().catch(err => {
+            if (!err.message.includes("router not registered")) {
```

Comment:
we don’t need this check anymore `router not registered`.
`getConnections ` will return an empty slice.

Reply:
fixed